app/src/main/java/ru/thstdio/aa2020/ui/list/FragmentMoviesList.kt

```diff
@@ -18,36 +18,24 @@ class FragmentMoviesList : FragmentNavigation(R.layout.fragment_movies_list) {
     }
 
     private val binding: FragmentMoviesListBinding by viewBinding()
-    private val exceptionHandler = CoroutineExceptionHandler { coroutineContext, exception ->
-        Log.e("Error", "CoroutineExceptionHandler got $exception in $coroutineContext")
+    private val appContext: Context
```

Comment:
private val appContext: Context
	         get() = requireActivity().applicationContext
	         
	         to global Fragment extension?

Reply:
fix